testcontainers/core/container.py

```diff
@@ -32,6 +33,10 @@ def with_exposed_ports(self, *ports) -> 'DockerContainer':
             self.ports[port] = None
         return self
 
+    def with_kargs(self, **kargs) -> 'DockerContainer':
+        self._kargs = kargs
```

Comment:
Is replacing the entire dictionary the desired behaviour? E.g. the current implementation does not allow the keyword arguments to be updated.